targets/esp8266/js2c.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
```

Comment:
It seems that this file is present in both targets. Do we really need it twice? Is it somehow target dependent?